UPGRADING.md

```diff
@@ -10,6 +10,33 @@ Next.js 9's dynamic routes are **automatically configured on [Now](https://zeit.
 
 You can read more about [Dynamic Routing here](https://github.com/zeit/next.js#dynamic-routing).
 
+#### Check your Custom <App> (`pages/_app.js`)
+
+If you previously copied the [Custom `<App>`](https://nextjs.org/docs#custom-app) example, you may be able to remove your `getInitialProps`.
+
+Removing `getInitialProps` from `pages/_app.js` (when possible) is important to leverage new Next.js features!
+
+The following `getInitialProps` does nothing and may be removed:
+
+```js
+class MyApp extends App {
+  // Remove me, I do nothing!
```

Comment:
I feel like this is not clear enough, you might want to comment out the getInitialProps.